www/js/app.js

```diff
@@ -1610,7 +1624,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                 $("#searchingArticles").hide();
                 alert("Error finding random article.");
             } else {
-                if (dirEntry.namespace === 'A') {
+                if (dirEntry.getMimetype() === 'text/html' || dirEntry.namespace === 'A') {
```

Comment:
Alternative here would be to test only `^text`, i.e.

`if (/^text\b/i.test(dirEntry.getMimetype() || dirEntry.namespace === 'A')`.

Reply:
This doesn't work because it catches text/css as well. We should keep the fallback and revise when we suppport titlePtrList v1.